src/dtw.jl

```diff
@@ -12,6 +12,56 @@ function nanopore_dtw(x::Vector{Float32}, y::Vector{Float32},
     return dtw_bhattacharyya(kmerdists1, kmerdists2)
 end
 
+
+function multi_nanopore_dtw(x::Vector{Vector{Float32}},
+     kmerpath::String = "models/r9.4_70bps.u_to_t_rna.5mer.template.model")
+     kmers = loadkmers(kmerpath)
+     kmerdists::Array{Array{kmerdist}} = []
+     for i in x
+        push!(kmerdists, kmerdist_from_changepoints(i, detect_change_points(i, kmers)))
+     end
+     seqlengths::Array{Int64} = []
+     for s in kmerdists
+         push!(seqlengths, length(s))
+     end
+     D = multi_pairwise_bhattacharyya(kmerdists, seqlengths)
+     return multi_trackback_bhattacharyya(D, seqlengths)
+end
+
+"""
+temporary function to graphically test the multi nanopore dtw function
+"""
```

Comment:
does it plot all the sequences that were mapped and their alignment?
if 3 sequences go in are 3 sequences on the plot?

Reply:
No, it was just a test function that i made to plot 2 of them, so that i could check that multi_nanopore_dtw worked